docs/Style-Guide.md

```diff
@@ -0,0 +1,180 @@
+
+**Example**
+
+Less Like this: we can use the following plugins.
+
+More like this: You can use the following plugins.
+
+> According to [Wikipedia](#), ***You*** is usually a second person pronoun. Also, used to refer to an indeterminate person, as a more common alternative to a very formal indefinite pronoun.
+
+## When to avoid second person "you" as the pronoun
+
+One of the main rules of formal writing such as references, synthesis and also in API documentation is to avoid second person ("you"). It should not directly address the user even when the word "you" is almost unavoidable when writing, but there are ways to avoid second person. Below are some alternative examples you can use to evade them. 
+
+**Example**
+
+Less Like this: You can use the following recommendation as an example.
+
+More like this: As an example, The following are recommendations to use.
+
+To view a documented example, visit the [Decorators.md](https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Decorators.md) page on Github. 
+
+
+### Avoid writing condescending languages.
+
+Condescending languages are words that include:
+
+* Just 
+* Easy
+* Simply
+* Basically
+* Obviously
+
+The reader may not find it easy to use the Fastify's framework and plugins, so avoid words that make it sound simple or easy. Not everyone who reads the documentation has the same level of understanding.
+
+
+### Starting with a verb
+
+Mostly start your description with a verb, which makes it simple and precise for the reader to follow. Use present tense more often, it's easier to read and understand than the past or future tense.
+
+**Example**
+
+ Less like this: There is a need for Nodejs to be installed before you can be able to use fastify.
+
+ More like this: Install Nodejs to make use of fastify.
+
+### Grammatical moods 
+
+Grammatical moods is a great way to express your writing, don't want to sound too bossy while making a direct statement. Know when to switch between indicative, imperative, and Subjunctive moods.
+
+
+**Indicative** - Use when making a factual statement or question. 
+
+Example: Since there is no testing framework available, "fastify recommend ways to write tests".
+
+**Imperative** - Use when giving instructions, actions, commands, or when you write your headings.
+
+Example: Install dependencies before starting application development mode.
+
+
+**Subjunctive** -  Use when making suggestions, hypothesis or non-factual statement.
+
+Example: Reading the documentation on our website is recommended to get comprehensive knowledge on the framework plugin.
+
+### Use **active** voice instead of **passive**
+
+Using active voice is a more compact and direct way of conveying your documentation.
+
+**Example**
+
+Passive: The dependencies of the npm package is installed by nodeJs.
+
+Active: NodeJs installs the npm package dependencies.
```

Comment:
Node.js does not do either of these things. `npm` is the package manager that installs dependencies.

Reply:
That is correct. The example was only to give a clear difference between active and passive voice.

Thanks for indicating the validity of my example, will put more effort into those as well.